### PR TITLE
3.29.0 changelog.

### PR DESCRIPTION
**Pull request by ndmckinley**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+
+FEATURES:
+* **New Data Source:** `google_cloud_identity_group_memberships` ([#6712](https://github.com/terraform-providers/terraform-provider-google/pull/6712))
+* **New Data Source:** `google_cloud_identity_groups` ([#6712](https://github.com/terraform-providers/terraform-provider-google/pull/6712))
+* **New Resource:** `google_kms_key_ring_import_job` ([#6682](https://github.com/terraform-providers/terraform-provider-google/pull/6682))
+* **New Resource:** `google_folder_iam_audit_config` ([#6708](https://github.com/terraform-providers/terraform-provider-google/pull/6708))
+
+IMPROVEMENTS:
+* bigquery: Added `"HOUR"` option for `google_bigquery_table` time partitioning (`type`) ([#6702](https://github.com/terraform-providers/terraform-provider-google/pull/6702))
+* bigquery: Add support for BigQuery hourly time partitioning  ([#6675](https://github.com/terraform-providers/terraform-provider-google/pull/6675))
```
Comment (slevenick) [inline CHANGELOG.md:14]:
```suggestion
* bigquery: Added support for BigQuery hourly time partitioning  ([#6675](https://github.com/terraform-providers/terraform-provider-google/pull/6675))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,26 @@
 ## 3.29.0 (Unreleased)
+NOTES:
+* added the `https://www.googleapis.com/auth/cloud-identity` scope to the provider by default ([#6681](https://github.com/terraform-providers/terraform-provider-google/pull/6681))
+* `google_app_engine_*_version`'s `service` field is required; previously it would have passed validation but failed on apply if it were absent. ([#6720](https://github.com/terraform-providers/terraform-provider-google/pull/6720))
+
+FEATURES:
+* **New Data Source:** `google_cloud_identity_group_memberships` ([#6712](https://github.com/terraform-providers/terraform-provider-google/pull/6712))
+* **New Data Source:** `google_cloud_identity_groups` ([#6712](https://github.com/terraform-providers/terraform-provider-google/pull/6712))
```
Comment (megan07) [inline CHANGELOG.md:8]: Sorry, I must have forgot to put beta-only on the data source's changelog entries, but the two cloud_identity data sources should just be beta-only.

Comment (ndmckinley) [reply CHANGELOG.md]: Thanks, done!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,24 @@
...
+
+FEATURES:
+* **New Resource:** `google_kms_key_ring_import_job` ([#6682](https://github.com/terraform-providers/terraform-provider-google/pull/6682))
+* **New Resource:** `google_folder_iam_audit_config` ([#6708](https://github.com/terraform-providers/terraform-provider-google/pull/6708))
+
+IMPROVEMENTS:
+* bigquery: Added `"HOUR"` option for `google_bigquery_table` time partitioning (`type`) ([#6702](https://github.com/terraform-providers/terraform-provider-google/pull/6702))
+* bigquery: Added support for BigQuery hourly time partitioning  ([#6675](https://github.com/terraform-providers/terraform-provider-google/pull/6675))
+* compute: Added `mode` to `google_compute_region_autoscaler` `autoscaling_policy` ([#6685](https://github.com/terraform-providers/terraform-provider-google/pull/6685))
+* compute: Added `scale_down_control ` to `google_compute_region_autoscaler` `autoscaling_policy` (beta only) ([#6685](https://github.com/terraform-providers/terraform-provider-google/pull/6685))
```
Comment (danawillow) [inline CHANGELOG.md:14]: looks like this is beta-only

**Pull request merged**